crates/pallet-domains/src/domain_registry.rs

```diff
@@ -268,9 +283,9 @@ mod tests {
                 Err(Error::InsufficientFund)
             );
             // Set enough fund to creator
-            <Test as Config>::Currency::make_free_balance_be(
+            Balances::make_free_balance_be(
                 &creator,
-                <Test as Config>::DomainInstantiationDeposit::get(),
+                <Test as Config>::DomainInstantiationDeposit::get() + 1,
```

Comment:
Why +1?

Reply:
We expect the account to atleast have existential deposit after locking funds. In tests, the existential deposit is `1`. Hence, minting the DomainDeposit + Existential deposit

Reply:
Should this be some `ExistentialDeposit const`? At least a comment.